examples/ex_cpu.c

```diff
@@ -0,0 +1,92 @@
+/* An example showing the use of al_get_cpu_count() and al_get_ram_size(). */
```

Comment:
There is one way to get log output on Android, and that's ALLEGRO_DEBUG but it only works with debug builds of Allegro.